### PR TITLE
Create fee add feature done button

### PR DESCRIPTION
**Pull request by huynhngocminh2511**

### CODE REVIEW

File: ClubManagement/ClubManagement/Activities/Base/CreateOrEditEventActivity.cs [hunk 1]
```diff
@@ -212,21 +215,14 @@ protected void Done(object sender, EventArgs e)
...
                 if (!isEdit)
                 {
-
-					this.ShowMessage(Resource.String.create_event_success);
-
-                    var intent = new Intent(this, typeof(EventDetailActivity));
-                    intent.PutExtra("EventDetail", eventDetail);
-
-                    StartActivity(intent);
+                    Toast.MakeText(this, Resource.String.create_event_success, ToastLength.Short).Show();
```
Comment (voquanghoa) [inline ClubManagement/ClubManagement/Activities/Base/CreateOrEditEventActivity.cs]: Replace 
```C#
Toast.MakeText(this, Resource.String.create_event_success, ToastLength.Short).Show();
```
By
```C#
this.ShowMessage(Resource.String.create_event_success);
```

File: ClubManagement/ClubManagement/Activities/CreateFeeActivity.cs [hunk 2]
```diff
@@ -10,6 +12,43 @@ namespace ClubManagement.Activities
...
     {
+        [InjectOnClick(Resource.Id.btnDone)]
+        private void Done(object sender, EventArgs e)
+        {
+            if (string.IsNullOrEmpty(edtDescription.Text) ||
+                string.IsNullOrEmpty(edtChooseFeeGroup.Text) ||
+                string.IsNullOrEmpty(edtDeadline.Text) ||
+                !long.TryParse(edtAmount.Text, out long amount) || amount == 0)
+            {
+                Toast.MakeText(this, GetString(Resource.String.please_fill_all_fields), ToastLength.Short).Show();
```
Comment (voquanghoa) [inline ClubManagement/ClubManagement/Activities/CreateFeeActivity.cs]: Use `this.ShowMessage(Resource.String.please_fill_all_fields)` please

File: ClubManagement/ClubManagement/Activities/Base/CreateOrEditEventActivity.cs (hunk 1 above)
Comment (huynhngocminh2511) [reply ClubManagement/ClubManagement/Activities/Base/CreateOrEditEventActivity.cs]: Done

File: ClubManagement/ClubManagement/Activities/CreateFeeActivity.cs (hunk 2 above)
Comment (huynhngocminh2511) [reply ClubManagement/ClubManagement/Activities/CreateFeeActivity.cs]: Done

**Pull request merged**